core/src/mindustry/logic/LExecutor.java

```diff
@@ -747,7 +750,8 @@ public void run(LExecutor exec){
             }
         }
 
-        void find(Ranged b, float range, int sortDir, Team team){
+        void find(Ranged b, float range, int sortDir, Team team, int idx){
+            ObjectMap<Float, Unit> units = new ObjectMap<>();
```

Comment:
This violates the contribution guidelines in two ways:
1. Boxed `Float` used.
2. Large allocation in main loop.

Reply:
Should be fixed now.